cpp/src/io/parquet/writer_impl.cu

```diff
@@ -1332,7 +1335,7 @@ void writer::impl::write(table_view const& table)
   for (uint32_t b = 0, r = 0; b < (uint32_t)batch_list.size(); b++) {
     uint8_t* bfr   = static_cast<uint8_t*>(uncomp_bfr.data());
     uint8_t* bfr_c = static_cast<uint8_t*>(comp_bfr.data());
-    for (uint32_t j = 0; j < batch_list[b]; j++, r++) {
+    for (size_type j = 0; j < batch_list[b]; j++, r++) {
       for (int i = 0; i < num_columns; i++) {
```

Comment:
For consistency with other loops iterating over numbers of columns, use `size_type`.
```suggestion
      for (size_type i = 0; i < num_columns; i++) {
```